File: socialuni-embed-demo-ui/src/model/violateWord/KeywordsDetailVO.ts
import Message from '@/model/Message'
import keywordsTriggerDetailVO from '@/model/violateWord/KeywordsTriggerDetailVO'
import keywordsVO from '@/model/violateWord/KeywordsVO'

export default class KeywordsDetailVO {
  // db的keywords
  tempKeywords: keywordsVO
  // 实时计算的keywords
  keywords: keywordsVO

  triggerDetails: keywordsTriggerDetailVO[]
  vioTriggerDetails: keywordsTriggerDetailVO[]
}

File: socialuni-server/socialuni-admin-sdk/src/main/java/com/socialuni/social/admin/model/KeywordsDetailVO.java
package com.socialuni.social.admin.model;

import com.socialuni.social.content.dao.DO.KeywordsDO;
import com.socialuni.social.content.dao.DO.KeywordsTriggerDetailDO;
import lombok.Data;

import java.util.List;

@Data
public class KeywordsDetailVO {
    //db的keywords
    private KeywordsDO tempKeywords;
    //实时计算的keywords
    private KeywordsDO keywords;

    private List<KeywordsTriggerDetailDO> triggerDetails;
    private List<KeywordsTriggerDetailDO> vioTriggerDetails;
}
